src/index.js

```diff
@@ -4,7 +4,10 @@ import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+// TODO: this hardcoded data will depend by the app language
+const locale = navigator.language.toLowerCase().split(/[_-]+/)[0] || 'en';
```

Comment:
Did you think how will you get this language from user data, since it's outside app component?

Reply:
This does not avoid the fact that maybe you get `loading...` message and then the app starts working in spanish, if it were the user language. But then the loading message is only temporary, so I guess is fine.